scripts/benchmark_migration.py

```diff
@@ -174,10 +172,9 @@ def main(
     models = find_models(module)
     model_rows: dict[type[Model], int] = {}
     for model in models:
-        rows = session.query(model).count()
+        rows = db.session.query(model).count()
         print(f"- {model.__name__} ({rows} rows in table {model.__tablename__})")
         model_rows[model] = rows
-    session.close()
```

Comment:
There is no need to explicitly close the session as this is handled by Flask-SQLAlchemy when the session is [torn down](https://github.com/pallets-eco/flask-sqlalchemy/blob/main/src/flask_sqlalchemy/extension.py#L441-L448)—be that at the end of request or when a script/shell terminates.